pennylane/vqe/vqe.py

```diff
@@ -89,6 +89,7 @@ class Hamiltonian:
 
         Note that this issue also arises when calling the ``simplify()`` method.
     """
+    num_params = 0
```

Comment:
wait, how come this is needed? Also, shouldn't it be `num_params=1`?

Reply:
This will make the circuit drawer work :) I added a comment and a drawer test.

Reply:
❤️